Xdr isolation and un-isolation warning on disconnected endpoint (#14751)
demisto/content: Packs/CortexXDR/ReleaseNotes/3_0_28.md

#### Integrations
##### Palo Alto Networks Cortex XDR - Investigation and Response

> - Added an argument for the **xdr-isolate-endpoint** and **xdr-unisolate-endpoint** commands whether or not to return an error when the endpoint is disconnected.

Reviewer: ```suggestion
- Added the *suppress_disconnected_endpoint_error* argument for the ***xdr-isolate-endpoint*** and the ***xdr-unisolate-endpoint*** commands which dictates whether to return an error when the endpoint is disconnected.
```
